src/Extend/ApiController.php

```diff
@@ -228,10 +298,10 @@ public function setSort(array $sort, $callback = null)
      * To force load the relationship, both levels have to be specified,
      * example: ['relation', 'relation.subRelation'].
      *
-     * @param string|array
+     * @param string[]
```

Comment:
the name of the parameter is missing, is this intentional? (`$relations`)

The implementation below seems to be designed for string parameter as well. Otherwise if it's just array, the type-hint can be added to the method and the cast to array can be removed.

Reply:
ah yeah 🤦🏼‍♂️ not sure how I ended up doing that.